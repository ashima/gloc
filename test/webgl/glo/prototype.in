void f();
int p(int);

void f() { p(0); }

struct s { int k; } g() { return s(10); }

s n = g(20);

void h(struct q { int a; } k);
struct s2 { int k; } j();

void main() { }
