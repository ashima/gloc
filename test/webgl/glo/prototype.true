{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [ "p" ],
      "outsym": [ "main", "f", "g", "s", "n" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nvoid f();\nint p(int);\n\nvoid f() { p(0); }\n\nstruct s { int k; } g() { return s(10); }\n\ns n = s(20);\n\nvoid main() { }"
    }
  ],
  "linkmap": {}
}
