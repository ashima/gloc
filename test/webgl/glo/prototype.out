{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "s2", "s", "p", "n", "main", "j", "h", "g", "f" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nvoid f();\nint p(int);\n\nvoid f() { p(0); }\n\nstruct s { int k; } g() { return s(10); }\n\ns n = g(20);\n\nvoid h(struct q { int a; } k);\nstruct s2 { int k; } j();\n\nvoid main() { }"
    }
  ],
  "linkmap": {}
}
